modin/engines/ray/pandas_on_ray/frame/partition_manager.py

```diff
@@ -220,6 +355,35 @@ def _apply_func_to_list_of_partitions(cls, func, partitions, **kwargs):
     def apply_func_to_select_indices(
         cls, axis, partitions, func, indices, keep_remaining=False
     ):
+        """
+        Apply a `func` to select indices of `partitions`.
+
+        Parameters
+        ----------
+        axis : {0, 1}
+            Axis to apply the `func` over.
+        partitions : np.ndarray
+            The partitions to which the `func` will apply.
+        func : callable
+            The function to apply to these indices of partitions.
+        indices : dict
+            The indices to apply the function to.
+        keep_remaining : bool, default: False
+            Whether or not to keep the other partitions. Some operations
+            may want to drop the remaining partitions and keep
+            only the results.
+
+        Returns
+        -------
+        np.ndarray
+            A NumPy array with partitions.
+
+        Notes
+        -----
+        Your internal function must take a kwarg `internal_indices` for
+        this to work correctly. This prevents information leakage of the
+        internal index to the external representation.
+        """
         return super(PandasOnRayFrameManager, cls).apply_func_to_select_indices(
             axis, partitions, func, indices, keep_remaining=keep_remaining
         )
```

Comment:
I wonder what is the point of declaring a function which _only_ calls a parent's version of itself...

Reply:
Probably, part of methods of this class makes simple calls of parent-class methods, because methods in this class is wrapped by progress bar.

Reply:
In this case they should be wrapped a little bit differently, but that's out of scope of documentation...